lib/3.6/cfe_internal.cf

```diff
@@ -234,3 +234,63 @@ bundle agent cfe_internal_cleanup_agent_reports
         delete => tidy,
         handle => "cf_cleanup_agent_reports_$(report_files)";
 }
+
```

Comment:
Can we put this into a new file so that we can load it only for the hub?

cfe_internal_enterprise_hub.cf perhaps?
This way we can load it only for the enterprise hub. (will have to be separate work to do that but moving this there would make that a bit easier).
You can simply add the new file to `inputs` in `bundle common stdlib` which is found in `lib/{3.6,3.7}/stdlib.cf`. For 3.5 you will need to add the file to `inputs` under the `cfengine_3_5` class guard in `bundle common cfengine_stdlib` in `promises.cf`